keps/sig-scheduling/2891-simplified-config/README.md

```diff
@@ -0,0 +1,553 @@
+multi-point extensions, especially for out-of-tree plugins. However the scheduler is capable of writing its
+internal config to a file usable by the user post-downgrade
+
+### Version Skew Strategy
+
+N/A - this does not affect other components
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [ ] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name:
+  - Components depending on the feature gate:
+- [x] Other
+  - Describe the mechanism: This will be a new field in the config which encapsulates a
+  subset of existing fields. Setting it implies enablement, leaving it unset does not.
+  - Will enabling / disabling the feature require downtime of the control
+    plane? Momentary downtime for the scheduler to load a new config, but this is not
+    specific to the feature (any config change requires a re-deployment of the scheduler)
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled). No
+
+###### Does enabling the feature change any default behavior?
+
+No
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+The field option can be unset and the scheduler re-deployed with existing config options.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+N/A
+
+###### Are there any tests for feature enablement/disablement?
+
+Scheduler unit tests will confirm translation and functioning of the config options as intended
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Misconfigured plugins could cause the scheduler to CrashLoop as it fails to initialize the framework.
+
+This does not have any effect on running workloads.
+
+###### What specific metrics should inform a rollback?
+
+Scheduler CrashLoops could indicate a failure to properly initialize the plugin registry
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+This will be manually tested
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+This feature has no functional change to workloads. The scheduler's functionality can be
+measured by its throughput and startup time.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [x] Other (treat as last resort)
+  - Details: Pods are scheduled as expected indicates functioning kube-scheduler
```

Comment:
NodeName being assigned?
Pod not being marked as unschedulable?

Reply:
This is tough to explicitly define, because pods could still be marked as unschedulable (or no NodeName assigned) in a functioning scheduler. Though I will add these as indicators since a high volume of them is suspect

Reply:
which means that people can also look at the `schedule_attempts_total` metric :)

Reply:
Thanks Aldo! I'll add that too